### PR TITLE
Dh home page UI

### PR DESCRIPTION
**Pull request by Dimo-Hristov**
Генериране на home page според дизайна. Всяка секция е във отделен компонент. Инсталирах Swiper за за слайдването на предстоящите събития, трябва да се инсталира от всички след мърджа. Също така рефактурирах листа със евенти, заместих hrml елементите със на ionic.

### CODE REVIEW

File: src/app/home/upcoming-events/upcoming-events.component.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<div *ngIf="slides.length > 0" class="a">
```
Comment (Getsov) [inline src/app/home/upcoming-events/upcoming-events.component.html]: Use more descriptive classes, avoid “a / b / c”

File: src/app/home/upcoming-events/upcoming-events.component.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  <swiper-container
+    class="b"
```
Comment (Getsov) [inline src/app/home/upcoming-events/upcoming-events.component.html]: Same here for the classes

File: src/app/home/upcoming-events/upcoming-events.component.html [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    [slidesPerView]="1.2"
+    [spaceBetween]="20"
+    [autoplay]="true"
+    [pagination]="{ clickable: true, dynamicBullets: true }"
+  >
+    <swiper-slide *ngFor="let slide of slides" class="c">
```
Comment (Getsov) [inline src/app/home/upcoming-events/upcoming-events.component.html]: Same here for the classes

File: src/app/home/upcoming-events/upcoming-events.component.ts [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+import { register } from 'swiper/element/bundle';
+import { IonicSlides } from '@ionic/angular';
+import { CommonModule } from '@angular/common';
+register();
+
+@Component({
+  selector: 'app-upcoming-events',
+  templateUrl: './upcoming-events.component.html',
+  styleUrls: ['./upcoming-events.component.scss'],
+  standalone: true,
```
Comment (Getsov) [inline src/app/home/upcoming-events/upcoming-events.component.ts]: Why using standalone components? We agreed not to use them, so we must be consistent

File: src/app/home/header/header.component.html [hunk 5]
```diff
@@ -0,0 +1,13 @@
+<ion-header class="home-header">
```
Comment (Getsov) [inline src/app/home/header/header.component.html]: Shouldn’t this be separated in different shared header (only part of it) component for the whole app?

File: src/app/home/home.page.ts [hunk 6]
```diff
@@ -8,42 +8,61 @@ import { Component, OnInit } from '@angular/core';
 export class HomePage implements OnInit {
   constructor() {}
 
-  ngOnInit() {}
+  slides: any[] = [];
+  eventsType: any[] = [];
+
+  ngOnInit() {
+    this.slides = [
```
Comment (Getsov) [inline src/app/home/home.page.ts]: Are the slides only going to have images in them? Without any info? And don’t forget the heart functionality for every slide (to like it), if it’s going to be implemented later, add a comment with TODO for it.
Avoid using common variable names. Isn’t this upcomingEvents ?

**Review by Getsov — CHANGES_REQUESTED** [review]
Also, the only R logo is missing

File: src/app/home/home.page.ts (hunk 6 above)
Comment (Dimo-Hristov) [reply src/app/home/home.page.ts]: done

File: src/app/home/header/header.component.html (hunk 5 above)
Comment (Dimo-Hristov) [reply src/app/home/header/header.component.html]: agree

File: src/app/home/upcoming-events/upcoming-events.component.ts (hunk 4 above)
Comment (Dimo-Hristov) [reply src/app/home/upcoming-events/upcoming-events.component.ts]: Yes, my falt. I was looking one demo with standalone component

File: src/app/home/upcoming-events/upcoming-events.component.html (hunk 3 above)
Comment (Dimo-Hristov) [reply src/app/home/upcoming-events/upcoming-events.component.html]: This is some serious season why i will never use "temporary" names for classes.

File: src/app/home/upcoming-events/upcoming-events.component.html (hunk 2 above)
Comment (Dimo-Hristov) [reply src/app/home/upcoming-events/upcoming-events.component.html]: done

File: src/app/home/upcoming-events/upcoming-events.component.html (hunk 1 above)
Comment (Dimo-Hristov) [reply src/app/home/upcoming-events/upcoming-events.component.html]: done

**Review by Getsov — APPROVED** [review]

**Pull request merged**